lib/srv/forward/sshserver.go

```diff
@@ -830,8 +831,8 @@ func (s *Server) handleDirectTCPIPRequest(ctx context.Context, ch ssh.Channel, r
 	}
 	scx.RemoteClient = s.remoteClient
 	scx.ChannelType = teleport.ChanDirectTCPIP
-	scx.SrcAddr = fmt.Sprintf("%v:%d", req.Orig, req.OrigPort)
-	scx.DstAddr = fmt.Sprintf("%v:%d", req.Host, req.Port)
+	scx.SrcAddr = net.JoinHostPort(req.Orig, strconv.Itoa(int(req.OrigPort)))
+	scx.DstAddr = net.JoinHostPort(req.Host, strconv.Itoa(int(req.Port)))
```

Comment:
I was wondering if we need to do `strconv.Itoa(int(…))` here but it looks like we do this in `lib/srv/legular/sshserver.go` as well so I suppose that's fine.

Reply:
Unfortunately we do because those are uint32's so we have to either convert it to int or `strconv.FormatUint(uint64(...), 10)`.